lookout/style/format/model.py

```diff
@@ -41,8 +42,20 @@ def dump(self) -> str:
         result = io.StringIO()
         result.write(super().dump())
         for lang, rules in sorted(self._rules_by_lang.items()):
-            result.write("\n\n# %s\n%s" % (lang, rules))
-        return result.getvalue()
+            result.write("\n\n# %s\n%s\n" % (lang, rules))
+            try:
+                for ds in ("train", "test"):
+                    print("## %s" % ds, file=result)
+                    print("PPCR: %f" % self[lang].classification_report[ds]["ppcr"], file=result)
+                    for r in ("report", "report_full"):
+                        print("### %s" % r, file=result)
+                        for avg in ("macro", "micro", "weighted"):
```

Comment:
Actually, we use only `micro avg` and there is no reason to store `macro` and `weighted` and dump it after. I can fix this in your PR because it's me who forget to exclude them from the model.

Reply:
Sorry, I was wrong. We use this info in individual reports. I think we can keep it.

Example: 

### Classification report

PPCR: 0.995

| Class | Precision | Recall | Full Recall | F1-score | Full F1-score | Support | Full Support | PPCR |
|------:|:----------|:-------|:------------|:---------|:---------|:--------|:-------------|:-----|
| `∅` | 0.993| 0.994| 0.993| 0.994| 0.993| 154185| 154384| 0.999 |
| `␣` | 0.963| 0.983| 0.983| 0.973| 0.973| 51116| 51116| 1.000 |
| `'` | 0.992| 1.000| 0.983| 0.996| 0.988| 22864| 23255| 0.983 |
| `␣'` | 1.000| 0.998| 0.998| 0.999| 0.999| 8351| 8351| 1.000 |
| `⏎␣⁺␣⁺` | 0.954| 0.941| 0.941| 0.948| 0.948| 7381| 7381| 1.000 |
| `⏎` | 0.958| 0.890| 0.890| 0.923| 0.923| 6483| 6483| 1.000 |
| `⏎␣⁻␣⁻` | 0.935| 0.916| 0.916| 0.925| 0.925| 5842| 5842| 1.000 |
| `⏎⏎` | 0.983| 0.818| 0.818| 0.893| 0.893| 2964| 2965| 1.000 |
| `'␣` | 0.000| 0.000| 0.000| 0.000| 0.000| 109| 269| 0.405 |
| `'⏎␣⁻␣⁻` | 0.000| 0.000| 0.000| 0.000| 0.000| 44| 286| 0.154 |
| `'⏎` | 0.000| 0.000| 0.000| 0.000| 0.000| 6| 220| 0.027 |
| `"` | 0.000| 0.000| 0.000| 0.000| 0.000| 0| 0| 0.000 |
| `micro avg` | 0.984| 0.984| 0.979| 0.984| 0.982| 259345| 260552| 0.995 |
| `macro avg` | 0.648| 0.628| 0.627| 0.638| 0.637| 259345| 260552| 0.995 |
| `weighted avg` | 0.983| 0.984| 0.979| 0.984| 0.980| 259345| 260552| 0.995 |